client/signup/jetpack-connect/main-wrapper.jsx

```diff
@@ -13,8 +13,9 @@ export default React.createClass( {
 	displayName: 'JetpackConnectMainWrapper',
 
 	render() {
+		const wrapperClassName = this.props.isWide ? 'jetpack-connect__main-wide' : 'jetpack-connect__main';
```

Comment:
Since this component is meant to be used in several other components, we should probably use PropTypes to make sure we're getting a bool.

So, we could do:

```
propTypes: {
    isWide: React.PropTypes.bool
},

getDefaultProps() {
    return {
        isWide: false
    }
},
```